wagtail/admin/templates/wagtailadmin/shared/side_panels/includes/status/locked.html

```diff
@@ -65,20 +69,18 @@
             {% endif %}
         {% endif %}
 
-        {% with icon_name='lock-open' %}
+        {% with icon_name='lock-open' help_text=help_text|capfirst %}
```

Comment:
This ensures that even if the sentence starts with `{{ model_name }}` in other languages, it will still be properly capitalised.